### PR TITLE
Improve homepage

### PR DESCRIPTION
**Pull request by nancruz**
In this commit:
* All URLs in homepage now open in a new tab
* Added url for the "Como Votar" button
* Added disclaimer before Party List

### CODE REVIEW

**Review by luisiul359 — COMMENTED** [review]
Have you updated social media links to open in a new tab?

**Comment by nancruz** [comment]
Nice catch @luisiul359 . I didn't

File: app/assets/stylesheets/home.scss [hunk 1]
```diff
@@ -80,3 +78,8 @@
 .home-warning-footer-wrapper {
     padding-bottom: 40px;
 }
+
+.home-parties-list_disclaimer {
+    text-align: center;
```
Comment (inesrosete) [inline app/assets/stylesheets/home.scss:85]: Text is not align. I was able to do it using `width: '600px', marginLeft: 'auto', marginRight: 'auto'`

Comment (nancruz) [reply app/assets/stylesheets/home.scss:85]: The disclaimer is aligned to the center.

image

**Review by luisiul359 — APPROVED** [review]
LGTM, but you have conflicts

**Pull request merged**